Community Groups information added

idvoretskyi opened the pull request:
Signed-off-by: Ihor Dvoretskyi <ihor@linux.com>

idvoretskyi commented:
cc @caniszczyk @amye FYI - ready for your review, please take a look.

Let me know if anything is missing.

amye commented:
Things we'll want to delete later: 'how to apply to be a meetup' but that's for ~December

idvoretskyi commented:
@amye links are correct (and updated), I've also updated the wording there - https://github.com/cncf/communitygroups/pull/17/commits/41fe8a950ece3bb0ecb79d13a78ef1d91c29906d
